interface/src/index.html

```diff
@@ -56,41 +56,41 @@
         <div class="collapse navbar-collapse" collapse="menuCollapsed">
           <ul class="nav navbar-nav">
             <li ui-sref-active="active">
-              <a href ui-sref="home">
+              <a ui-sref="home">
```

Comment:
There are a couple more in the `experimentDetail.tpl.html` and `sampleDetail.tpl.html` pages, but I'll need to get those pages to handle state parameters properly before fixing that. Making an issue...